.circleci/config.yml

```diff
@@ -711,6 +721,25 @@ jobs:
       - persist_to_workspace: *artifacts_executables
       - gitter_notify_failure_unless_pr
 
+  b_ubu_static_arm:
+    <<: *base_ubuntu2004_arm_large
```

Comment:
I do not know the average time as I have not access to CircleCI and have done no benchmark. That said, the _large_ instance is the fastest one in the free tier. https://circleci.com/docs/using-arm

Reply:
The CI is public so you should be able to view the results (you may have to login with a github account though).

Anyway, yeah, if the CI checks in this PR are not running under the solidity org then the instance sizes may be limited. In that case we'll have to adjust it ourselves. I'm going to keep this comment open so that we don't forget.